cnf-tests/testsuites/e2esuite/xt_u32/xt_u32.go

```diff
@@ -63,7 +69,7 @@ var _ = Describe("xt_u32", func() {
 		var testNode string
 
 		BeforeEach(func() {
-			if !hasNonCnfWorkers {
+			if !hasNonCnfWorkers || isSingleNode {
```

Comment:
I think is better to create a different skip for the single node with a better message (test need multiple nodes or something like that)